input/scss/_global.scss

```diff
@@ -1,10 +1,18 @@
 // Global styling for this template
+html {
+  height: 100%;
+}
 body {
+  height: 100%;
   font-size: 20px;
   color: $gray-900;
   @include serif-font;
 }
 
+header.masthead + .container {
+  min-height: 100%;
+}
```

Comment:
okay for now, we can revisit this in future if we want to separate out the footer from main container.